pkgs/top-level/python-packages.nix

```diff
@@ -2477,7 +2477,17 @@ in modules // {
     };
   };
 
-  boto3 = buildPythonPackage rec {
+  boto3 = let
+    # Version >=1.3,<1.4 is required by boto3.
+    botocore-1_3 = self.botocore.override rec {
+      name = "botocore-${version}";
+      version = "1.3.30";
+      src = pkgs.fetchurl {
+        url = "https://pypi.python.org/packages/6c/79/5d3abfb9fffe4a8cb0a113f088c1012743cf40db805aaac625b2e3cf2111/botocore-1.3.30.tar.gz";
+        md5 = "0bbccf3385a0e509e5a674ddb2d27752";
```

Comment:
Please don't use md5! We're trying to deprecate it :) There's very little reason not to use sha256 these days. MD5 is horribly broken and SHA1 is moderately broken.